lib/utils/exception_handler.dart

```diff
@@ -264,31 +275,31 @@ class ExceptionHandler {
     };
   }
 
-  static void showError(String error, {int? delayInSeconds}) async {
+  static Future<void> showError(String error, {int? delayInSeconds}) async {
     if (_hasError) {
       return;
     }
     _hasError = true;
-
     if (delayInSeconds != null) {
       Future.delayed(Duration(seconds: delayInSeconds), () => _showCopyPopup(error));
       return;
     }
 
-    WidgetsBinding.instance.addPostFrameCallback(
-      (_) async => _showCopyPopup(error),
-    );
+    await Future.delayed(Duration.zero);
+    await _showCopyPopup(error);
   }
 
   static Future<void> _showCopyPopup(String content) async {
     if (navigatorKey.currentContext != null) {
+      final seeds = await secureStorageShared.read(key: "corruptedWalletsSeed");
+      await secureStorageShared.delete(key: "corruptedWalletsSeed");
       final shouldCopy = await showPopUp<bool?>(
         context: navigatorKey.currentContext!,
         builder: (context) {
           return AlertWithTwoActions(
             isDividerExist: true,
             alertTitle: S.of(context).error,
-            alertContent: content,
+            alertContent: content+"\n"+(seeds??""),
```

Comment:
here I believe it would be duplicated, as the content is the error message, which contains the seeds as well